force-app/main/default/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -1002,26 +990,26 @@ export default class GeFormRenderer extends LightningElement{
      * @param sectionsList - Array, all sections
      * @returns {boolean} - true if error message was generated, false if otherwise
      */
-    getDonorTypeValidationError(fieldWrapper, sectionsList) {
+    getDonorTypeValidationError(sectionsList) {
```

Comment:
Do we need this additional function? Could we move the boolean block for `isError` in this function into `isDonorTypeInvalid()` since that seems to be the intention of `isDonorTypeInvalid()`.  We should probably then extract the `if (isError)` block into its own function since it has doing something else other than checking the validity of the donor type and seems to have the responsibility of displaying the error message. That function could be named something like `displayDonorTypeError()`. Then, that function could be called in the `isFormValid()` function before we return the `false` value. This way we aren't also calling a display function inside a boolean function whose responsibility is only to check the validity.

Thoughts?

Reply:
Agree with extracting the `if (isError) { ... }` block.